13-tool-usage-cycle-tracking/README.md

```diff
@@ -246,6 +242,34 @@ These environment variables are set in Notehub, either per-device, per-fleet or
 
 Please see our tutorial [Understanding Environment Variables](https://dev.blues.io/guides-and-tutorials/notecard-guides/understanding-environment-variables/) for a fuller description of how to set environment variables in Notehub.
 
+
+### Vibration Monitoring and Alerts
+
+The Notecard accelerometer is used to sense vibration. Tool usage can be inferred from the measured vibration using environment variables that define the expected vibration characteristics. The vibration environment variables define thresholds of vibration for when the tool is inactive and active.
+
+* `alert_off_vibration`: the *maximum* vibration expected when the tool is *inactive*. Minimum value is `0`, which is also the default. Must be less than or equal to `alert_under_vibration`.
```

Comment:
I’m kind of curious what “vibration” is measured in. I don’t think I’d know what value I’d need to provide here.

Reply:
Ah I see you have them defined later on. Maybe just link up the sections.

Reply:
Done.